src/screens/Connectors/ConnectorUIUtils/PaymentMethod.res

```diff
@@ -165,7 +165,7 @@ module CardRenderer = {
                   <CheckBoxIcon isSelected={isSelected(value)} />
                 </div>
               </AddDataAttributes>
-              <p className=p2RegularTextStyle>
+              <p className=p2RegularTextStyle onClick={_ => removeOrAddMethods(value)}>
```

Comment:
please move the <AddDataAttributes> to cover the test case

Reply:
@JeevaRamu0104, made the changes accordingly